assets/static/js/main.js

```diff
@@ -0,0 +1,247 @@
+(function () {
+  let clockTimer
+  let weatherTimer
+  let refreshTimer
+  let tz
+  let currentWeatherId
+
+  const assetsPath = '/static'
+  const imagesPath = `${assetsPath}/images`
+  const iconsPath = `${assetsPath}/images/icons`
+  /**
+   * Utility Functions
+   */
+  const getDayString = (day) => {
+    const days = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday']
+    return days[day]
+  }
+
+  const getMonthString = (month) => {
+    const months = ['Jan', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December']
```

Comment:
Nitpicking but please change `Jan` to `January` to keep it consistent.

Reply:
@vpetersson Done